test/lan_udev_auto_test.rb

```diff
@@ -4,9 +4,12 @@
 
 require "yast"
 
-module Yast
+include Yast
 
-  Yast.import "UI"
+Yast.import "UI"
+Yast.import "LanUdevAuto"
+
+describe '#getDeviceName' do
```

Comment:
This is wrong, you should always specify whole method call like `LanUdevAuto.getDeviceName` or if it is part of include then e.g. describe "Yast::Udev#getDeviceName"